app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-<h1>Display Page For User</h1>
+<h1>Display Page For A Particular User</h1>
+
+<%= render 'user' %>
+<div class='bio'>
+  <h2>Bio<h2>
+  <p><%= @user.bio %></p>
+</div>
+
+<% @user.last_three_posts.each do |post| %>
+<div class='post'>
+  <br>
+
+  <%= link_to  post.title, user_post_path(@user, post.id)  %>
+
+  <p><%= post.text %></p>
+  <span class='counter'>
+    <span>Comments: <%= post.comments_counter %></span>
+    <span>, Likes: <%= post.likes_counter %></span>
```

Comment:
### **Optional**
- [ ] Please modify your comment and like counter to display `0` when no comment or like is recorded.
### **Try it this way**
```
<span>Comments: <%= post.comments_counter || 0 %>,</span> 
<span>Likes: <%= post.likes_counter || 0 %></span>
```

Reply:
Thank you